st2client/tests/base.py

```diff
@@ -86,6 +86,8 @@ def setUp(self):
             if var in os.environ:
                 del os.environ[var]
 
+        os.environ['ST2_SKIP_CLI_CONFIG'] = '1'
```

Comment:
>Without this option, you need to "temporary" remove or move the config.
Yeah, that's more explicit and will be the thing with least surprises. People will forget to reset env variables.